faster_whisper/utils.py

```diff
@@ -21,6 +21,7 @@
     "large-v1": "guillaumekln/faster-whisper-large-v1",
     "large-v2": "guillaumekln/faster-whisper-large-v2",
     "large": "guillaumekln/faster-whisper-large-v2",
+    "large-v3": "flyingleafe/faster-whisper-large-v3",
```

Comment:
Did you convert this after the [pr fixing alignment heads was included in ctranslate2](https://github.com/OpenNMT/CTranslate2/pull/1530)? and if not, what is the expected impact?